src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -95,6 +95,11 @@
     private static String formatBirthDate(final String dob) {
         DateFormat tdsFormat = new SimpleDateFormat("MMddyyyy");
         DateFormat trtFormat = new SimpleDateFormat("yyyy-MM-dd");
+
```

Comment:
I think this should be the first line in the method and I think it should do a `if(StringUtils.isBlank(dob)) { return default}` since that will be the result of the format call on line 104